### PR TITLE
Release v0.3.0

### PR DESCRIPTION
**Pull request by QuantumDancer**

### CODE REVIEW

**Review by dirksammel — COMMENTED** [review]
What about the changelog at the bottom of the `rpm/*spec` files?

**Comment by QuantumDancer** [comment]
Ah right, completely forgot that :sweat_smile: 
I've now also update the changelog in the RPM spec files

File: rpm/auditor-priority-plugin.spec [hunk 1]
```diff
@@ -29,6 +29,8 @@ rm -rf $RPM_BUILD_ROOT
 %{_bindir}/%{name}
 
 %changelog
+* Thu Nov 17 2023 Benjamin Rottler <benjamin.rottler@physik.uni-freiburg.de> - 0.3.0
```
Comment (dirksammel) [inline rpm/auditor-priority-plugin.spec:32]: Today is `Fri Nov 17 2023`. Same comment for all `spec` files.

**Review by dirksammel — CHANGES_REQUESTED** [review]

File: rpm/auditor-priority-plugin.spec (hunk 1 above)
Comment (QuantumDancer) [reply rpm/auditor-priority-plugin.spec]: Right, fixed. I also udpated the `Changelog.md`, because there the release date was 2023-11-16

**Review by dirksammel — APPROVED** [review]

**Pull request merged**